pytorch_lightning/loggers/comet.py

```diff
@@ -312,3 +313,7 @@ def __getstate__(self):
         # needed later
         state["_experiment"] = None
         return state
+
+    def log_graph(self, model: LightningModule, input_array=None) -> None:
```

Comment:
Tensorboard logger has a `log_graph` arg in `__init__`, that indicates if the graph should be automatically added during `.fit`. I would propose we also add that arg to this logger now.

Reply:
That sounds good to me. Should it default to `False` like the TB logger does? Does the trainer always call `logger.log_graph(model)` during `.fit` or how does this happen automatically? It also seems from the TB logger that if you set `log_graph=False` at init, then even if the user calls `logger.log_graph(model)` explicitly, nothing will be done. Is that a desired behavior? It seems counterintuitive to me.

Reply:
Default to `False`
The trainer calls `logger.log_graph(model)` here (always during training):
https://github.com/PyTorchLightning/pytorch-lightning/blob/062800aa99cff6eb82838b355374305d9433507e/pytorch_lightning/trainer/training_loop.py#L156-L160
and it is then determined by the `log_graph` flag if something should be done. 
I will agree that it is counter intuative for the user that `log_graph` arg also need to be enabled if I manually wants to log the graph.